lib/robots/dor_repo/was_crawl_preassembly/end_was_crawl_preassembly.rb

```diff
@@ -9,8 +9,7 @@ def initialize
         end
 
         def perform_work
-          current_version = object_client.version.current
-          workflow_service.create_workflow_by_name(druid, 'accessionWF', version: current_version)
+          object_client.version.close(description: 'Closed by WAS crawl preassembly robot to initiate accessioning') # Starts the accessionWF by default
```

Comment:
Here and below: No need to provide a description.

This is a description of the version (not the closing operation). It can optionally be provided when closing, but isn't necessary.

Reply:
@justinlittman cool, done.